src/projects/project-pull-request-list-item.html

```diff
@@ -8,35 +8,44 @@
 <dom-module id="project-pull-request-list-item">
   <template>
     <style>
-      .ci-status-failed {
+      .margin.indicator.true {
         background-color: var(--primary-action-red);
       }
-      .ci-status-failed .passed {
+      .icon-failed {
+        fill: var(--primary-action-red);
+      }
+      .icon-failed.passed {
         display: none;
       }
-      .ci-status-passed {
-        background-color: var(--primary-action-green);
+      .icon-passed {
+        fill: var(--pr-status-color);
```

Comment:
I think this should be green instead. It is now gray, I prefer green to indicate a good CI.